### ISSUE TITLE
Still working ? Works on Chrome, Not postman/package

### ISSUE THREAD

**Issue by EskAere** [viralvaghela/flutter_insta#24]
Is the package still working ? 

I did the request using the package, I got http instead of JSON. Also tried to do it in Postman, still got HTTP error. 
The only way to make the request work is via Chrome. Is Instagram detecting that the request is done programmatically and then blocks it ?

**Comment by viralvaghela** [comment]
Hi @EskAere , This will be fixed soon, working on it #25

**Comment by viralvaghela** [comment]
Hello @EskAere, the issue has been fixed and updated!🎉  Check it out now and let me know if still have issues.

**Comment by EskAere** [comment]
Hello @viralvaghela. 

I hoped you had a better solution than mine ... I had already updated the url on my side. 
Have you tried it on your side ? Because it is still not working on mobile for me

**Comment by viralvaghela** [comment]
Yes it's working,did you update the package version in pubspec file?

**Comment by EskAere** [comment]
I did. 
But as I said, I had already done the changes from the latest update. I re-tried with your instagram account, both on mobile (real phone and emulator) and Postman again. Only got Http errors (no-js not-logged-in client-root touch most of the time). No idea why it only works on browser.
